index.js

```diff
@@ -63,8 +63,9 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        file[name] = included[filename];
+        file[name].contents = included[filename].contents;
```

Comment:
Don't need to change the `.contents` explicitly, since `file[name] = included[filename]` does it. Don't need the `.contents` line.